### PR TITLE
fix typos in constants.ts, Form.tsx

### PR DESCRIPTION
**Pull request by HanJaeJoon**
### Reasons for making this change
fix typos
- RJSF_ADDITONAL_PROPERTIES_FLAG => RJSF_ADDITIONAL_PROPERTIES_FLAG
- acceptcharset => acceptCharset

### Checklist

- [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://rjsf-team.github.io/react-jsonschema-form/docs/contributing) of the Markdown text I've added
- [x] **I'm adding or updating code**
  - [ ] I've added and/or updated tests. I've run `npm run test:update` to update snapshots, if needed.
  - [ ] I've updated [docs](https://rjsf-team.github.io/react-jsonschema-form/docs) if needed
  - [x] I've updated the [changelog](https://github.com/rjsf-team/react-jsonschema-form/blob/main/CHANGELOG.md) with a description of the PR
- [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: packages/utils/src/constants.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const PROPERTIES_KEY = 'properties';
 export const REQUIRED_KEY = 'required';
 export const SUBMIT_BTN_OPTIONS_KEY = 'submitButtonOptions';
 export const REF_KEY = '$ref';
-export const RJSF_ADDITONAL_PROPERTIES_FLAG = '__rjsf_additionalProperties';
+export const RJSF_ADDITIONAL_PROPERTIES_FLAG = '__rjsf_additionalProperties';
```
Comment (heath-freenome) [inline packages/utils/src/constants.ts:26]: For backwards compatibility, we'll need to keep the constants and deprecate them
```suggestion
/**
 * @deprecated Replace with correctly spelled constant `RJSF_ADDITIONAL_PROPERTIES_FLAG`
 */
export const RJSF_ADDITIONAL_PROPERTIES_FLAG = '__rjsf_additionalProperties';
export const RJSF_ADDITIONAL_PROPERTIES_FLAG = '__rjsf_additionalProperties';
```

File: packages/core/src/components/Form.tsx [hunk 2]
```diff
@@ -112,7 +112,7 @@ export interface FormProps<T = any, S extends StrictRJSFSchema = RJSFSchema, F e
   onFocus?: (id: string, data: any) => void;
   // <form /> HTML attributes
   /** The value of this prop will be passed to the `accept-charset` HTML attribute on the form */
-  acceptcharset?: string;
+  acceptCharset?: string;
```
Comment (heath-freenome) [inline packages/core/src/components/Form.tsx:115]: For backwards compatibility we'll need to keep both for now:
```suggestion
/** The value of this prop will be passed to the `accept-charset` HTML attribute on the form
 * @deprecated replaced with `acceptCharset` which will supercede this value if both are specified
 */
  acceptcharset?: string;
/** The value of this prop will be passed to the `accept-charset` HTML attribute on the form */
  acceptCharset?: string;
```

File: packages/core/src/components/Form.tsx [hunk 3]
```diff
@@ -870,7 +870,7 @@ export default class Form<
       action,
       autoComplete,
       enctype,
-      acceptcharset,
+      acceptCharset,
```
Comment (heath-freenome) [inline packages/core/src/components/Form.tsx:873]:
```suggestion
      acceptcharset,
      acceptCharset,
```

File: packages/core/src/components/Form.tsx [hunk 4]
```diff
@@ -905,7 +905,7 @@ export default class Form<
         action={action}
         autoComplete={autoComplete}
         encType={enctype}
-        acceptCharset={acceptcharset}
+        acceptCharset={acceptCharset}
```
Comment (heath-freenome) [inline packages/core/src/components/Form.tsx:908]:
```suggestion
        acceptCharset={acceptCharset || acceptcharset}
```

File: packages/core/src/components/Form.tsx (hunk 2 above)
Comment (heath-freenome) [reply packages/core/src/components/Form.tsx:115]: I don't see this change applied yet

Comment (HanJaeJoon) [reply packages/core/src/components/Form.tsx:120]: Sorry, I missed it. I applied it.

**Review by heath-freenome — APPROVED** [review]

**Pull request merged**